karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
I'm still listening and considering. Thanks for pushing me to learn more. So far these are all solved problems, with the tools I was already using, I just hadn't configured them, yet.

That said, if this is an easier solution, I'm open to it. Just glancing at the config though, it looked slightly more complicated and verbose than the npm scripts I planned to use. Kindof like Grunt but specifically for testing. My impression so far is that it solves problems we don't have.


## Watch

You can get the same watch features with webpack (which you also added) or watch, or [https://github.com/substack/watchify](watchify).


## Multiple browsers

The best way to do automated browser testing is with a cloud service provider such as [Sauce Labs](https://saucelabs.com) or [BrowserStack](https://www.browserstack.com/). Testling (the cli tool) is also a near-zero config solution for local browser testing that works perfectly with npm scripts.

## Integration and Support

How does it make life easier to integrate with tools like Jenkins, TravisCI, CircleCI, etc...? What does it give us that the npm scripts don't?